Media/PlateDetectedEvent/schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Media/PlateDetectedEvent/schema.json",
+  "title": "FIWARE - Media / Plate detected Event",
+  "description": "Event raised by a PlateDetectorFilter when a plate is found in the data streamed.",
+  "type": "object",
+  "allOf": [
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons" },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "MediaEvent"
+          ],
+          "description": "NGSI Entity type"
+        },
+        "eventType": {
+          "type": "string"
```

Comment:
most of the properties in this schema are already defined in MediaEvent. i would used definitions there, and import the schema here.

Reply:
Done!